### ISSUE TITLE
place of softmax function 

### ISSUE THREAD

**Issue by sdjsngs** [bkong999/COVNet#28]
Thanks for your work， but  i have noticed that in main.py you use the softmax function after the loss.backward(),  this is  a unusual
operation， in a classification task , the softmax function often  between the fc layer ( in model class ) and  loss.bacward() ,   
Sincerely， hope you to replay 
thank you

**Comment by doge-ac-cn** [comment]
I think it's right.
metric(prediction, label.long()) is  torch.nn.CrossEntropyLoss() , it contains softmax.
And the second softmax is just to calculate the average accuracy of all the slices .
